### PR TITLE
repokitteh: adding warning about draft PRS

### PR DESCRIPTION
**Pull request by alyssawilk**
Hey @itayd is there a better way to test this works than submitting and creating a draft PR? :-)

### CODE REVIEW

**Comment by itayd** [comment]
unfortunately not yet :( i don't see any warning in this pr though?

**Comment by phlax** [comment]
@alyssawilk im a bit confused by this pr - as @itayd said i dont see any warning - and im struggling to understand the connection to repokitteh as this change is to the github notifier

**Comment by alyssawilk** [comment]
whoops, I had two branches named draft in 2 different trees and pushed the wrong one. 
Given it's minimal code I'll just merge the two - sorry about the force push but it's like 6 lines of overwrite so hopefully not bad =P

File: ci/repokitteh/modules/newcontributor.star [hunk 1]
```diff
@@ -8,26 +8,52 @@ In the meantime, please take a look at the [contribution guidelines](https://git
 
 """
 
+DRAFT_MESSAGE = """
```
Comment (phlax) [inline ci/repokitteh/modules/newcontributor.star:11]: i think we should perhaps just include this in the main message

the reason being that until recently i didnt discover how to *create* a draft pr - so habitually created a non-draft and then marked it so

Comment (phlax) [reply ci/repokitteh/modules/newcontributor.star:11]: hmmm - altho i guess this also sends a message to non-new-contributors - which is kinda confusing given the name of this module

Comment (alyssawilk) [reply ci/repokitteh/modules/newcontributor.star:11]: yeah, if someone has contributed a few times and then does a draft I think they should get the warning.  I don't think there is a hook for "has ever done a draft before" so it's all or nothing.
moved the module for clarity

**Review by phlax — DISMISSED** [review]
lgtm - we'll have to "first song sound check" with it i reckon as no easy way to test repokitteh

**Review by phlax — APPROVED** [review]
lgtm, thanks

**Pull request merged**